From c0a24da25fed6503ae1eafc92d6d91277091039d Mon Sep 17 00:00:00 2001
From: Jackie Huang <jackie.huang@windriver.com>
Date: Mon, 28 Oct 2019 16:20:54 +0800
Subject: [PATCH] requirements.txt: add urllib3<1.24

Fixed the issue:
/usr/local/lib/python3.4/site-packages/requests/__init__.py:91:
RequestsDependencyWarning: urllib3 (1.25.6) or chardet (3.0.4) doesn't
match a supported version!
  RequestsDependencyWarning)

Signed-off-by: Jackie Huang <jackie.huang@windriver.com>
---
 requirements.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/requirements.txt b/requirements.txt
index 64f8999..534b542 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -10,3 +10,4 @@ cryptography>=2.3, <=2.4.2
 yamlreader==3.0.4
 pluggy>=0.5, <0.7
 packaging==17.1
+urllib3<1.24
-- 
2.7.4

